docs/js/navbar-search.js

```diff
@@ -1,19 +1,24 @@
-var defaultVersion = '6.x';
-
 (function() {
-  var versionFromUrl = window.location.pathname.match(/^\/docs\/(\d+\.x)/);
-  var version = versionFromUrl ? versionFromUrl[1] : defaultVersion;
+  const defaultVersion = '6.x';
```

Comment:
Mental note to self to check why `defaultVersion = '6.x'` here

Reply:
Turns out it doesn't really matter what the default version is. I'll remove that code in this PR.

Reply:
probably because of a merge of the 6.x branch, or just plainly forgotten to change with the release of 7.0 (https://github.com/Automattic/mongoose/blob/791ab3e7e7929564340ab4c56ade2c7cbb1fe5fc/docs/js/navbar-search.js#L1)